_plugin-as-service/ml-commons.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: ML-Commons Plugin as a Service
+has_children: false
+nav_order: 10
+---
+
+# ML-Commons Plugin as a Service 🚀
+
+## 🏗 Multi-Tenancy Enhancements
+We have introduced **multi-tenancy** support for:
+- **Connectors**
+- **Model Groups**
+- **Models**
+- **Agents**
+- **Tasks**
+- **Config APIs**
+
+> ✅ **Upcoming Updates:** We will continue expanding multi-tenancy to additional APIs incrementally.
+
+---
+
+## ⚙️ ML Commons Multi-Tenancy Settings
+To enable **multi-tenancy** in ML Commons, set the following cluster setting:
+
+```yaml
+plugins.ml_commons.multi_tenancy_enabled: true
+```
+
+### **🔧 Values**
+| Setting | Default Value | Allowed Values |
+|---------|--------------|----------------|
+| `plugins.ml_commons.multi_tenancy_enabled` | `true` | `true` or `false` |
```

Comment:
Hmm... each plugin initializes their own client but it seems silly to define these settings in multiple locations (and I haven't yet for Flow Framework, just using system properties instead).  We should put some thought into how to reduce/avoid duplication.

Reply:
yeah up for the later discussion.